### DISCUSSION TITLE
TinyBMS

### DISCUSSION THREAD

**Discussion by gyrex** [Louisvdw/dbus-serialbattery#557] [New BMS requests]
I love your project! Could you please add support for TinyBMS: https://www.energusps.com/shop/product/tiny-bms-s516-150a-750a-36?category=4

There's a communication protocol PDF here too: https://www.energusps.com/web/binary/saveas?filename_field=datas_fname&field=datas&model=ir.attachment&id=21208

Many thanks!! :)

**Comment by tenteb** [comment]
Good morning, thank you very much for your excellent work.
I have a victron system with multiplus and cerbo gx and 4 mercedes batteries in parallel on a bus bar and each one with a Tiny BMS. You could contact me and give me a price to be able to integrate these bms. I'm waiting. Greetings and thanks.
https://enepaq.com/wp-content/uploads/2022/07/TinyBMS_Communication_Protocols_Rev_C.pdf

**Comment by amarce** [comment]
Hi,

I woud love to see your driver working with Tiny BMS too. Can we help you somehow in order to make it possible?

Thanks a lot!
